lib/manageiq/automation_engine/engine.rb

```diff
@@ -5,9 +5,9 @@ module AutomationEngine
     class Engine < ::Rails::Engine
       isolate_namespace ManageIQ::AutomationEngine
 
-      config.autoload_paths << root.join("app/models/mixins")
-      config.autoload_paths << root.join("lib/miq_automation_engine")
-      config.autoload_paths << root.join("lib/miq_automation_engine/engine")
+      config.autoload_paths << root.join("app/models/mixins").to_s
+      config.autoload_paths << root.join("lib/miq_automation_engine").to_s
+      config.autoload_paths << root.join("lib/miq_automation_engine/engine").to_s
```

Comment:
Same as on the other PR, we probably want to put the link to the ruby bug for more information